web/flow/inspect.go

```diff
@@ -39,7 +40,12 @@ func handleInspect(ctx context.Context, rt *runtime.Runtime, r *inspectRequest)
 	var sa flows.SessionAssets
 	// if we have an org ID, create session assets to look for missing dependencies
 	if r.OrgID != models.NilOrgID {
-		oa, err := models.GetOrgAssetsWithRefresh(ctx, rt, r.OrgID, models.RefreshFields|models.RefreshGroups|models.RefreshFlows)
+		refreshFlags := models.RefreshFields | models.RefreshGroups | models.RefreshFlows
```

Comment:
What if by default this is the list of types that can be created on the fly by the editor? I think these are here because you might create a group on the fly in the editor and then a revision is saved and inspect is called immediately.

And then for imports we need a mode that includes the types that imports can create.

Reply:
I don't think imports can create labels, optins or topics.. no?

Reply:
Hmm maybe we just need to add `models.RefreshGlobals` and not bother passing a flag... I see `Flow.update_dependencies` creates those but I'm not seeing other places we create types on the fly

Reply:
Why `models.RefreshTriggers` ?

Reply:
I believe we need labels, optin, topics and triggers too

https://github.com/nyaruka/rapidpro/blob/be17482db738a7522cac0d2fad5045c9a3a6da70/temba/flows/models.py#L479

Reply:
Now they are ordered alphabetically